server/Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN cargo chef prepare --recipe-path recipe.json
 FROM chef AS build
 
 RUN apt-get update && apt-get install -y \
+        curl=7.* \
```

Comment:
Won't work as this is the build step, not the runtime step, though I see below that you did a different healthcheck command. Let me ask the team to take a look. Thanks!